crates/sui-core/src/test_utils.rs

```diff
@@ -310,7 +310,7 @@ pub fn make_transfer_object_transaction(
         object_ref,
         sender,
         gas_object,
-        gas_price * TEST_ONLY_GAS_UNIT_FOR_TRANSFER,
+        gas_price * TEST_ONLY_GAS_UNIT_FOR_TRANSFER * 10,
```

Comment:
is `transfer_object` this expensive?

Reply:
I do not know, but it did not pass without this